host/xtest/xtest_main.c

```diff
@@ -67,6 +71,9 @@ void usage(char *program)
 	printf("\t--aes-perf [opts]  AES performance testing tool (-h for usage)\n");
 #ifdef CFG_SECURE_DATA_PATH
 	printf("\t--sdp-basic [opts] Basic Secure Data Path test setup ('-h' for usage)\n");
+#endif
+#ifdef CFG_SECURE_KEY_SERVICES
+	printf("\t--sks [opts]       Secure key services test setup ('-h' for usage)\n");
```

Comment:
Why do we need yet another test suite, couldn't this be part of the regression suite instead?